src/Centreon/Domain/Monitoring/ResourceService.php

```diff
@@ -152,11 +152,13 @@ public function findResources(ResourceFilter $filter): array
      */
     public function enrichHostWithDetails(ResourceEntity $resource): void
     {
-        $downtimes = $this->monitoringRepository->findDowntimes(
-            $resource->getId(),
-            0
-        );
-        $resource->setDowntimes($downtimes);
+        if ($resource->getInDowntime() === true) {
```

Comment:
we'll need to rename later `getInDowtime` by `isInDowntime`

Reply:
Yes I did not take the time to do it as multiple impacts which are not that mandatory as 21.10 will reach EOL soon